corgi/collectors/brew.py

```diff
@@ -345,13 +345,17 @@ def get_container_build_data(self, build_id: int, build_info: dict) -> dict:
             # Legacy OSBS builds such as 1890187 copy source code into dist-git but specify where
             # the source code came from using the 'go' stanza in container.yaml
             # ref: https://osbs.readthedocs.io/en/osbs_ocp3/users.html#go
-            go_modules = []
-            if "go" in build_info["extra"]["image"]:
-                for module in build_info["extra"]["image"]["go"]["modules"]:
-                    if "module" in module:
-                        go_modules.append(module["module"])
-            if go_modules:
-                component["meta"]["upstream_go_modules"] = go_modules
+            # Handle case when "go" key is present but value is None
```

Comment:
perhaps I missed ... how is this related to inlinie styling and /docs endpoint ?